### ISSUE TITLE
dotf@1.3.0 is broken

### ISSUE THREAD

**Issue by PopGoesTheWza** [grant/dotf#10]
The npm does not contain neither the lib/index.js nor the lib/index.d.ts

**Comment by PopGoesTheWza** [comment]
PR merged

**Comment by grant** [comment]
`1.4.0` was published.

**Comment by PopGoesTheWza** [comment]
thanks @grant

**Comment by PopGoesTheWza** [comment]
@grant please check what got packaged in 1.4.0
Something is wrong. Did you run `build` and checked the output of `dryrun`?

**Comment by grant** [comment]
I checked the `package.json` and `lib/index.js`:
```
npm run dryrun

> dotf@1.4.0 dryrun /Users/timmerman/Documents/github/grant/dotf
> npm publish --dry-run

npm notice
npm notice 📦  dotf@1.4.0
npm notice === Tarball Contents ===
npm notice 1.0kB package.json
npm notice 1.1kB LICENSE
npm notice 1.1kB README.md
npm notice 465B  lib/index.d.ts
npm notice 1.6kB lib/index.js
npm notice === Tarball Details ===
npm notice name:          dotf
npm notice version:       1.4.0
npm notice package size:  2.6 kB
npm notice unpacked size: 5.3 kB
npm notice shasum:        22c1a9c781ea37d20194544e7aeac07cdc930442
npm notice integrity:     sha512-9qGysIs00RzHk[...]u1bvxKnciNp7w==
npm notice total files:   5
npm notice
+ dotf@1.4.0
```

**Comment by PopGoesTheWza** [comment]
@grant I don't know why/how, but something is wrong in 1.4.0
![image](https://user-images.githubusercontent.com/32041843/58346139-6e0ccd80-7e5a-11e9-87f8-317a07525fac.png)

**Comment by PopGoesTheWza** [comment]
@grant sorry for I have no practical experience of publishing package. Still do not hesitate to ping me if I can help in anything

**Comment by grant** [comment]
I'm just trying to figure out what is broken.

**Comment by PopGoesTheWza** [comment]
Looking at the content of 1.4.0, it is like the `.npmignore` is not properly accounted for... Which makes little sense when checking the `npm publish --dry-run` or `npm pack` results

**Comment by grant** [comment]
I think we could simplify the `npmignore`. See clasp:
https://github.com/google/clasp/blob/master/.npmignore

What about trying to publish in a test package before publishing the real package.

**Comment by PopGoesTheWza** [comment]
npm pop-dotf is fine

**Comment by PopGoesTheWza** [comment]
I simple changed the "nam" in package.json and did `npm publish` :/

**Comment by grant** [comment]
Hmmm...
So I think the current state of things is fixed. The latest published package for `dotf` is `1.2.0`:
```
npm dist-tag add dotf@1.2.0 latest
```
---
Verified this with a Google engineer.

Will look into publishing this later.

**Comment by PopGoesTheWza** [comment]
@grant agreed. Meanwhile, I'll test the pop-dotf@1.4.0 package with current clasp

No need to rush until next week :)

**Comment by PopGoesTheWza** [comment]
@grant perhaps you should consider unpublishing 1.3.0 and 1.4.0

Here is the documentation to do this 
https://docs.npmjs.com/unpublishing-packages-from-the-registry

**Comment by PopGoesTheWza** [comment]
@grant any plans on re-trying to fix `dotf` publish?

**Comment by PopGoesTheWza** [comment]
fixed with 1.4.1
